build_tools/install_iree_host_tools.sh

```diff
@@ -8,27 +8,78 @@
 
 # Bash script to build / install the IREE host tools.
 
-if [ "${VIRTUAL_ENV}" != "" ]; then
+if [[ $# -ne 0 && $# -ne 1 ]] ; then
+  echo "Usage: $0"
+  echo "   or: $0 -pip"
+  echo ""
+  echo "By default, the iree-dist installed is fetched and unpacked."
+  echo "In addition, IREE can be installed via pip, whereas additional"
+  echo "tools are installed from source."
+  exit 1
+fi
+
+
+# Check if pip should be used instead of fetching the iree-dist tarball
+
+if [[ $# -eq 1 && "$1" != "-pip" ]]; then
+  echo "$0: unrecognized option '${1}'"
+  exit 1
+fi
+
+if [[ "${VIRTUAL_ENV}" != "" ]]; then
   echo "$0 cannot be executed in a venv since it creates a new venv."
   exit 1;
 fi
 
+# Determine which version to install
+
 PATH_TO_SCRIPT="`dirname $0`"
 PATH_TO_REPO="`realpath ${PATH_TO_SCRIPT}/../`"
-IREE_VERSION="`sed -n 2p ${PATH_TO_REPO}/requirements.txt | sed 's/.*=//'`"
+
+if [[ $# -eq 0 ]] ; then
+  IREE_VERSION="`sed -n 's/.*dist-\(.*\)-linux-x86_64.tar.xz/\1/p' ${PATH_TO_REPO}/iree-release-link.txt`"
+else
+  IREE_VERSION="`sed -n 2p ${PATH_TO_REPO}/requirements.txt | sed 's/.*=//'`"
+fi
 
 echo "Installing IREE host tools candidate-${IREE_VERSION}"
 
+
+# Create the IREE_HOST_BINARY_ROOT directory
+
+mkdir -p build-iree-host-install-${IREE_VERSION}/bin
+ln -sfn build-iree-host-install-${IREE_VERSION} build-iree-host-install
+
+
+# Use the iree-dist tarball
+
+if [[ $# -eq 0 ]] ; then
+  python3 -m venv venv-iree
+  source venv-iree/bin/activate
+  pip3 install -r numpy Pillow
```

Comment:
The `-r` needs to be dropped here. Will adjust this together with incorporating your review suggestions.